### PR TITLE
Changes for the homepage

### PR DESCRIPTION
**Pull request by ajnsit**
Need to:
1. Fix the nav at the top of the page. Add a background color, and
possible a logo when the page is scrolled down.
2. Add an 'About' that should go to the top of the page. It should keep
the nav in view.
3. Fix the entries to have separate upcoming and past events sections
4. Add an FP Jobs section

### CODE REVIEW

**Review by ManasJayanth — COMMENTED** [review]
Got back browsersync. Let's keep it as it is very convenient. Fixed the navbar styles

File: index.html [hunk 1]
```diff
@@ -13,7 +13,7 @@
       class="light fixed top-1 left-1 z-50 p-2 block sm:hidden border-2 border-solid rounded">
       <span class="line"></span>
     </button>
-    <nav id="nav" class="hidden fixed flex flex-col justify-center text-xl sm:flex-row sm:justify-end h-full sm:h-auto top-0 left-0 right-0 bottom-0 sm:bottom-auto bg-gray-700 sm:bg-white w-full p-4 sm:text-right">
+    <nav id="nav" class="flex fixed hidden flex-col justify-center text-xl h-full top-0 left-0 right-0 bottom-0 bg-gray-700 p-4 sm:bg-white w-full sm:flex-row sm:justify-end sm:h-auto sm:bottom-auto sm:text-right">
```
Comment (utkarshkukreti) [inline index.html]: Try `hidden sm:flex` here and remove the two `nav` rules in the CSS.

PS: It doesn't make sense to have both `flex` and `hidden` class as they set the same property (`display`).

Comment (utkarshkukreti) [reply index.html]: And the default flex direction is `row` so you don't need `sm:flex-row`. Some other classes can also probably be removed but I'll have to run this in a browser to confirm.

**Comment by ManasJayanth** [comment]
I remember trying sm:flex but to of no use. Will try again. Is the nav
appearing fine for you?

**Comment by utkarshkukreti** [comment]
Ok, the JS is messing things up. It's toggling only `hidden` and not `flex`. I'll see if I can do something.

**Comment by utkarshkukreti** [comment]
Ok, try this out:

```diff
diff --git a/index.html b/index.html
index aebe915..30f9584 100644
--- a/index.html
+++ b/index.html
@@ -13,7 +13,7 @@
       class="light fixed top-1 left-1 z-50 p-2 block sm:hidden border-2 border-solid rounded">
       <span class="line"></span>
     </button>
-    <nav id="nav" class="flex fixed hidden flex-col justify-center text-xl h-full top-0 left-0 right-0 bottom-0 bg-gray-700 p-4 sm:bg-white w-full sm:flex-row sm:justify-end sm:h-auto sm:bottom-auto sm:text-right">
+    <nav id="nav" class="fixed hidden sm:flex flex-col justify-center text-xl h-full top-0 left-0 right-0 bottom-0 bg-gray-700 p-4 sm:bg-white w-full sm:flex-row sm:justify-end sm:h-auto sm:bottom-auto sm:text-right">
         <a href="#about">About</a>
         <a href="#upcoming">Upcoming Events</a>
         <a href="#past">Past Events</a>
@@ -223,6 +223,7 @@
           .querySelector("nav")
           .addEventListener("click", function() {
             document.querySelector("#nav").classList.toggle("hidden");
+            document.querySelector("#nav").classList.toggle("flex");
 	    navButton.classList.toggle("light");
           });
 	let navButton = 
@@ -231,6 +232,7 @@
         navButton.addEventListener("click", function() {
 	    navButton.classList.toggle("light");
             document.querySelector("#nav").classList.toggle("hidden");
+            document.querySelector("#nav").classList.toggle("flex");
           });
       });
     </script>
diff --git a/styles/app.css b/styles/app.css
index 336d4d6..436f018 100644
--- a/styles/app.css
+++ b/styles/app.css
@@ -60,17 +60,6 @@ nav a {
   }
 }
 
-nav {
-  display: none;
-}
-
-@screen sm {
-  nav {
-    display: flex !important;
-    flex-direction: row;
-  }
-}
-
 #nav-button:before,
 .line,
 #nav-button:after {
```

**Pull request merged**